spec/cypress/integration/admin/administrators/delete-administrator-user.js

```diff
@@ -12,10 +12,10 @@ describe("Admin user deleting another admin user", () => {
     cy.get("[data-test=edit-admin-link]").contains(adminUserName).click();
     cy.get("[data-test=delete-button]").click();
     // @todo replace with visual test
-    cy.get("main").should("contain", "Confirm deletion?");
+    cy.get("main").should("contain", "Do you want to delete this admin user?");
```

Comment:
Hehe, don't you wish we had visual tests that you would have to recreate instead of this? :D

Reply:
I wouldn't have needed to change it three times in this PR so far if they were visual tests because visual tests only need approving once! (and they're approved by pressing a button, not by having to modify a test file)

Reply:
Oh, I thought they work more like snapshot tests. Interesting, now I hope we get them even more.